settings.gradle.kts

```diff
@@ -5,18 +5,14 @@ pluginManagement {
         gradlePluginPortal()
     }
 
-    val toolsVersion = "0.10.2"
     val kotlinVersion = "1.5.21"
 
     plugins {
-        id("ru.mipt.npm.gradle.project") version toolsVersion
-        id("ru.mipt.npm.gradle.mpp") version toolsVersion
-        id("ru.mipt.npm.gradle.jvm") version toolsVersion
+        id("ru.mipt.npm.gradle.project") version "0.10.2"
         kotlin("multiplatform") version kotlinVersion
-        kotlin("jvm") version kotlinVersion
         kotlin("plugin.allopen") version kotlinVersion
         id("org.jetbrains.kotlinx.benchmark") version "0.3.1"
-        kotlin("jupyter.api") version "0.10.0-131-1"
+        kotlin("jupyter.api") version "0.10.0-173"
```

Comment:
Jupyter is now loaded in common plugin: https://github.com/mipt-npm/gradle-tools/blob/100f7cd55ff04b540caa7260078bb68fa76b8afe/build.gradle.kts#L33. No need to write it here

Reply:
OK

Reply:
use 
```kotlin
kscience{
  useJupyter()
}
```